tools/remote_build/rbe_common.bazelrc

```diff
@@ -91,4 +91,4 @@ build:ubsan --test_timeout=60,600,1800,3600
 # how to update the bazel toolchain for ubsan:
 # - check for the latest released version in https://github.com/bazelbuild/bazel-toolchains/tree/master/configs/experimental/ubuntu16_04_clang
 # - you might need to update the bazel_toolchains dependency in grpc_deps.bzl
-build:ubsan --crosstool_top=@bazel_toolchains//configs/experimental/ubuntu16_04_clang/1.3/bazel_0.29.1/ubsan:toolchain
+build:ubsan --crosstool_top=@bazel_toolchains//configs/experimental/ubuntu16_04_clang/1.3/bazel_1.0.0/ubsan:toolchain
```

Comment:
This line (both the new and the old version is tricky since it prevents us from upgrading the bazel toolchain to anything recent - see https://github.com/grpc/grpc/pull/25001#issuecomment-745559569 for that).
Looks like we'll have to spend some time figuring out how to run ubsan on RBE so that :
- we can re-enable UBSAN with absl
- we can upgrade our bazel toolchain